blockstore/splitstore/splitstore_warmup.go

```diff
@@ -44,11 +50,12 @@ func (s *SplitStore) warmup(curTs *types.TipSet) error {
 // objects are written in batches so as to minimize overhead.
 func (s *SplitStore) doWarmup(curTs *types.TipSet) error {
 	epoch := curTs.Height()
+	boundaryEpoch := epoch - WarmupBoundary
```

Comment:
This might underflow to negative in integration tests where we use the splitstore. Maybe we should add a Ceil?

Reply:
ah yes, good point -- I'll add a ceiling check.

Reply:
done.